### DISCUSSION TITLE
Build fails with "Provided Google Play service account JSON has invalid format" error

### DISCUSSION THREAD

**Discussion by miguelcmedeiros** [codemagic-ci-cd/codemagic-docs#1582] [Q&A] [answered]
I'm trying to run a build and it keeps failing with the following error message:
"Provided Google Play service account JSON has invalid format"

There's no other log in the build aside from this message. It seems that Codemagic runs a pre-run validation over the yaml file, and fails then.

I managed to run the build successfully last Friday and as far as I know, nothing changed since then.
Any idea what this might be?

**Build ID:** 643faafef3b40e9159605c7b

**Comment by FabianClaasenWhyellow** [comment]
I have the same issue when I make the environment variable containing the JSON secured. When it is not a secured variable it works just fine.

**Comment by himesh-cm** [comment]
I have notified the team and will keep you posted.

**Reply by himesh-cm** [reply, marked as answer]
The fix is live. Please try and let us know in case of any issues.

**Reply by miguelcmedeiros** [reply]
Awesome, thanks!!!
